tests/integration/test_orders_cli.py

```diff
@@ -417,25 +417,16 @@ def test_cli_orders_download_state(invoke, order_description, oid):
     assert 'order state (running) is not a final state.' in result.output
 
 
-# TODO: add tests of "create --pretty" (gh-491).
```

Comment:
What was the purpose of changing this test? Seems not relevant to the tickets.

Reply:
I was considering changing this test to test the inputs (json string, etc) and while I was in there I noticed this parameterization isn't used (id_string is never used and there is no logic to handle differences in expected_ids). I realized that the various inputs are all tested in the types module (for JSON type) so didn't need to actually add to this test but in the interest of cleaning as we go, I left the change to the test.